### PR TITLE
Add instructions for including the devloper workspace button in a README

### PR DESCRIPTION
**Pull request by tomgeorge**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?

Adds instructions for including the "Developer Workspace" button to a project `README`

### What issues does this PR fix or reference?

eclipse/che#16731

### CODE REVIEW

File: src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+sidebar: che_7_docs
+permalink: che-7/the-developer-workspace-button
+summary:
+---
+
+:page-liquid:
+:parent-context-of-the-developer-workspace-button: {context}
+
+[id="the-developer-workspace-button_{context}"]
+= The Developer Workspace Button
```
Comment (amisevsk) [inline src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc:14]: I think this title is only clear if you already know what you want -- maybe something like `Adding a markdown button for opening your repo in Che`?

File: src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+== Including the Developer Workspace Button in a `README`
+
+=== Procedure
+
+Add a link to your repository in the project `README`.  Using a GitHub repository as an example:
+
+[source,shell]
+----
+[![Contribute](https://che.openshift.io/factory/resources/factory-contribute.svg)](https://che.openshift.io/f?url=https://github.com/org/repository)
```
Comment (amisevsk) [inline src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc:37]: Perhaps we should use `che.openshift.io` as an example and list general instructions for adding the button? You should be able to replace `https://che.openshift.io` with whatever URI your Che server is deployed on.

Comment (tomgeorge) [reply src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc:37]: That’s correct.  This is an oversight on my part

**Review by MichalMaler — APPROVED** [review]

File: src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: The Developer Workspace Button
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/the-developer-workspace-button
```
Comment (l0rd) [inline src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc:6]: @tomgeorge I suspect that you need a trailing slash here.

Comment (tomgeorge) [reply src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc:6]: opened https://github.com/eclipse/che-docs/pull/1462

**Pull request merged**